### DISCUSSION TITLE
graphql compatability

### DISCUSSION THREAD

**Discussion by jamalalikhan** [rmosolgo/graphql-ruby#4489] [General]
Is graphql compatible with ruby 2.5 I am unable to install it.

**Comment by rmosolgo** [comment]
Maybe not! I have a CI job that runs on 2.4 and 2.7, but not 2.5. Why not share some details about what went wrong? That might enable someone to help debug the problem.

**Reply by jamalalikhan** [reply]
gem install was not working properly for me bundle add did the trick. thanks for the response. the latest version of graphql 2.0.22 is working fine with ruby 2.5.9
